### ISSUE TITLE
Understanding how many tiles are created: 

### ISSUE THREAD

**Issue by linker2424** [openseadragon/openseadragon#1340]
All,

Can anyone point me where in the documentation it show the algorithm for how many tiles are created?

For example:
If I have an image that’s 1031 x 1077, and I setup the source as a DZI, OSD will request at level 10 the following tiles:
0_0
0_1
0_2
1_0
1_1
1_2
2_0
2_1
2_2
3_0
3_1
3_2

How does it know to ask for only 12 tiles at this level and not 16? What if the aspect ratio was in reverse i.e., 1077 x 1031 would it ask for the tiles in a different order or a different amount of tiles? 
Thanks everyone!

**Comment by linker2424** [comment]
Is my wording to confusing?

**Comment by iangilman** [comment]
@linker2424 Yes, your question makes sense. We should definitely have more documentation around this. See https://github.com/openseadragon/site-build/issues/28 related issue.

For a general overview of the DZI format, you can see: 

https://msdn.microsoft.com/en-us/library/cc645077(VS.95).aspx

...but unfortunately it's not very detailed. Here's an attempt to add a little more detail:

https://github.com/openseadragon/openseadragon/wiki/The-DZI-File-Format

Please let me know what you think!

Regarding the specific image you referred to above, it depends on the tile size.

**Comment by iangilman** [comment]
By the way, it would be great to have some visual diagrams to go with that wiki page, if anyone wants to try their hand at that!
